flink-filesystems/flink-fs-hadoop-shaded/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,31 @@
+This project includes software developed at
```

Comment:
the apache template includes the module name

Reply:
This could be quite helpful when the notice files are being merged.

Reply:
Added.